cgp/population.py

```diff
@@ -128,3 +128,13 @@ def fitness_parents(self) -> List[Union[None, float]]:
             List of fitness values for all parents.
         """
         return [ind.fitness for ind in self._parents]
+
+    def reorder_genome(self) -> None:
+        """ Reorders the genome for all parents in the population
+
+        Returns
+        ---------
+        None
+        """
+        for parent in self.parents:
```

Comment:
as discussed this reorders the parents forever, rather than only before creating offsprings. it hence adds an additional (neutral) drift through the search space. could you check again please how this is handled in the original paper and (if we keep it like this) make sure to document it appropriately in the doc for the `do_reorder` argument?

Reply:
In the paper it says: "Reorder is used once each generation to shuffle the nodes of the parent’s genome". This is not fully explicit, but if you look at the source code: 
https://github.com/brianwgoldman/Analysis-of-CGPs-Mechanisms/blob/master/evolution.py#L591

Reorder is applied to the parent even if the parent is not replaced at the end of a generation.